src/functions/helpers.ts

```diff
@@ -126,6 +161,59 @@ export function toJsDate(value: string | number | boolean | null | undefined): D
   return numberToJsDate(toNumber(value));
 }
 
+export enum ABSOLUTE_RELATIVE_MODE {
+  ROW_ABSOLUTE_COL_ABSOLUTE = 1,
+  ROW_ABSOLUTE_COL_RELATIVE = 2,
+  ROW_RELATIVE_COL_ABSOLUTE = 3,
+  ROW_RELATIVE_COL_RELATIVE = 4,
+}
+
+export function toA1Notation(
+  rowNumber: number,
+  colNumber: number,
+  absoluteRelativeMode: number = ABSOLUTE_RELATIVE_MODE.ROW_ABSOLUTE_COL_ABSOLUTE
+): string {
+  const colLetter = numberToLetters(colNumber - 1);
+  let cellReference: string;
+  switch (absoluteRelativeMode) {
+    case ABSOLUTE_RELATIVE_MODE.ROW_ABSOLUTE_COL_RELATIVE:
+      cellReference = `${colLetter}$${rowNumber}`;
+      break;
+    case ABSOLUTE_RELATIVE_MODE.ROW_RELATIVE_COL_ABSOLUTE:
+      cellReference = `$${colLetter}${rowNumber}`;
+      break;
+    case ABSOLUTE_RELATIVE_MODE.ROW_RELATIVE_COL_RELATIVE:
+      cellReference = `${colLetter}${rowNumber}`;
+      break;
+    case ABSOLUTE_RELATIVE_MODE.ROW_ABSOLUTE_COL_ABSOLUTE:
+      cellReference = `$${colLetter}$${rowNumber}`;
+      break;
+    default:
+      throw new Error(expectNumberRangeError(1, 4, absoluteRelativeMode));
+  }
+  return cellReference;
+}
+
+export function fromA1NotationToR1C1Notation(A1Notation: string): string {
```

Comment:
Maybe a regexp is overkill here, let's avoid regexp. Maybe it would be better to compute R1C1 from rowNumber,  colNumber and absoluteRelativeMode too